inference/sources/fly/test/fly_example.json

```diff
@@ -0,0 +1,72 @@
+{
+  "total_items": 3,
```

Comment:
I don't see this in our api: https://github.com/vlab-research/fly/pull/111/files#diff-011281b666d6b1e9a47971f545727d534d43873392a339e9ded300eafcdf4530R98